erpnext/portal/product_configurator/utils.py

```diff
@@ -313,13 +313,20 @@ def get_items(filters=None, search=None):
 
 	search_condition = ''
 	if search:
+		# Default fields to search from
+		default_fields = {'name', 'item_name', 'description', 'item_group'}
+
+		# Get meta search fields
+		meta = frappe.get_meta("Item")
+		meta_fields = set(meta.get_search_fields())
+
+		# Join the meta fields and default fields set
+		search_fields = default_fields.union(meta_fields)
+
+		# Build or filters for query
 		search = '%{}%'.format(search)
-		or_filters = [
-			['name', 'like', search],
-			['item_name', 'like', search],
-			['description', 'like', search],
-			['item_group', 'like', search]
-		]
+		or_filters = [[field, 'like', search] for field in search_fields]
```

Comment:
We should not run LIKE statement in description field if there are huge numbers of item, the query becomes very slow. We also can't index description. Please refer https://github.com/frappe/erpnext/blob/develop/erpnext/controllers/queries.py#L174

Reply:
I'll add the condition mentioned in the link you mentioned

Reply:
Updated in both PRs